app/gear/local/local_impl.py

```diff
@@ -153,6 +153,72 @@ def get_user_by_id(self, user_id: int):
             self.log.log_error_message(e, self.module)
             return ResponseNOK(message="User cannot be retrieved.", code=202)
         return value
+    # Indicadores de chaco
+    def indicador_usuarios_activos(self) -> int:
+        try:
+            contador = self.db.query(model_user.id_user_status.like(1)).where(model_user.is_admin == 0).count()
```

Comment:
@ifdamianboden cuál es la idea de este indicador?

Si ejecuto el este endpoint me da un número de 237. Pero si ejecuto esta query, que supongo que es lo que se está haciendo en el código: `SELECT COUNT(id) FROM user WHERE id_user_status = 1 AND is_admin = 0;` me da 26.